host/src/PrelaunchApp/App.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
I see [this commit ](https://github.com/Azure/azure-functions-dotnet-worker/commit/52c54406d28f96dc52fd4872379d24c966904856#diff-8ab6a51b6f9f40b8125c8e8f0397522d3ad5ad9999bb5f94c05a92d26f328bc7L15)removed `net7.0` from this version list.

I did not remove that tfm from project file in this PR as I want to follow up to understand the reasoning behind that change first.